### ISSUE TITLE
 Multilingual Translation FR-EN, DE-EN 

### ISSUE THREAD

**Issue by Emelement** [pytorch/fairseq#2061]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
Hello, i'm using the  code availabe on https://github.com/pytorch/fairseq/tree/master/examples/translation in order to train a multilingual translator FR-EN, DE-EN i trained it using the dataset available in the code and the same hyper parameters however once loaded, the translations are not good at all. The sentences that i'm trying to translate are very domain specific can this be the source of the problem ?

#### Code
mkdir -p checkpoints/multilingual_transformer
CUDA_VISIBLE_DEVICES=0 fairseq-train data-bin/iwslt17.de_fr.en.bpe16k/ \
    --max-epoch 50 \
    --ddp-backend=no_c10d \
    --task multilingual_translation --lang-pairs de-en,fr-en \
    --arch multilingual_transformer_iwslt_de_en \
    --share-decoders --share-decoder-input-output-embed \
    --optimizer adam --adam-betas '(0.9, 0.98)' \
    --lr 0.0005 --lr-scheduler inverse_sqrt --min-lr '1e-09' \
    --warmup-updates 4000 --warmup-init-lr '1e-07' \
    --label-smoothing 0.1 --criterion label_smoothed_cross_entropy \
    --dropout 0.3 --weight-decay 0.0001 \
    --save-dir checkpoints/multilingual_transformer \
    --max-tokens 4000 \
    --update-freq 8

   

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by lematt1991** [comment]
How does the model do when translating sentences in the validation set?  This should give you a better sense of whether or not the model is bad or your sentences are too out of domain.

**Comment by Emelement** [comment]
Thanks for the response !
I'm getting 3.728 loss and 2.119 nll_loss for the validation set

**Comment by lematt1991** [comment]
I mean in terms of BLEU or even just manually inspecting the translations.  Do they seem sensible?

**Comment by Emelement** [comment]
i'm getting an error when i try to generate and score the test set with sacrebleu with the code on https://github.com/pytorch/fairseq/tree/master/examples/translation

The error i'm getting is sacreBLEU: Test set iwslt17 contains no sentence
